facebook.js

```diff
@@ -1,78 +1,148 @@
+var stad = chrome.i18n.getMessage("stand");
+var per = chrome.i18n.getMessage("per");
```

Comment:
Loading the data from file would be preferred as having all the maps in the same file would get out of hand quickly. On the other hand it will make it impossible for the extension to be used as a user script.